lms/djangoapps/support/static/support/jsx/entitlements/components/EntitlementForm/index.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, InputSelect, InputText, TextArea } from '@edx/paragon';
+
+
+class EntitlementForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const isReissue = props.entitlement !== null && props.entitlement !== undefined;
```

Comment:
If the `entitlement` `prop` is required then it should never be `null` / `undefined` right? (Are you seeing prop type warnings in the console?)

If it needs a sensible default, don't forget to add a `defaultProps`.